planning/development-strategy.md

```diff
@@ -1,39 +1,142 @@
-# Restaurants finder
+# Job - Advisor
+
+<!---minju edited--->
+
+Job Advisor website is for motivated students and job seekers to help them to
+kickstart their career in IT and also search their dream job without
+registration.
 
 ---
 
-<!-- ## User Story Dependencies
+## Setup
 
-[Story Dependency Diagram](https://excalidraw.com/)
+- [ ] Boilerplate HTML file
+- [ ] CSS file for header, program section, Main info and footer
+- [ ] CSS file is linked to HTML
 
 ---
 
-## WIREFRAME
+### Logo
+
+> As a User I want to see a logo so that I can identify >the website easily and
+> navigate to the home page from >any pages.
+
+- [ ] _There is a clear logo same as link to the home page in the header_.
+
+### logo: HTML
+
+- Header with the web site's logo.
 
-![wireframe]() -->
+### logo: CSS
+
+- styling to the top left of the main page.
 
 ---
 
-## 0.Setup
+### Navigation Bar
+
+> As a userI want to easily navigate the site
+
+- [ ] _There is a navigation bar with links to other pages of the site(About
```

Comment:
Nav bar is should have. Please remove from here. Development strategy is only for must-haves

Reply:
sure